.gitignore

```diff
@@ -13,3 +13,5 @@ coverage.html
 .glide
 .scBuildImage
 .init
+.generate*
+**/zz*.go
```

Comment:
I didn't agree to that :-)
I'm still waiting for a good reason for us to check it in. Yes I can think of one or two but none of them apply to this project yet.  I see no reason not to generate this like any other output file during the build process.

People have expressed concerns in the past with checking-in vendored code because someone may "accidentally" modify it - how is this any different?